docs/source/cuda/bindings.rst

```diff
@@ -4,28 +4,40 @@ CUDA Bindings
 Numba supports two bindings to the CUDA Driver APIs: its own internal bindings
 based on ctypes, and the official `NVIDIA CUDA Python bindings
 <https://nvidia.github.io/cuda-python/>`_. Functionality is equivalent between
-the two bindings, with two exceptions:
-
-* the NVIDIA bindings presently do not support Per-Thread Default Streams
-  (PTDS), and an exception will be raised on import if PTDS is enabled along
-  with the NVIDIA bindings.
-* The profiling APIs are not available with the NVIDIA bindings.
+the two bindings.
 
 The internal bindings are used by default. If the NVIDIA bindings are installed,
 then they can be used by setting the environment variable
 ``NUMBA_CUDA_USE_NVIDIA_BINDING`` to ``1`` prior to the import of Numba. Once
 Numba has been imported, the selected binding cannot be changed.
 
 
+Per-Thread Default Streams
+--------------------------
+
+Responsibility for handling Per-Thread Default Streams (PTDS) is delegated to
+the NVIDIA bindings when they are in use. To use PTDS with the NVIDIA bindings,
+set the environment variable ``CUDA_PYTHON_CUDA_PER_THREAD_DEFAULT_STREAM`` to
+``1`` instead of Numba's environmnent variable
+:envvar:`NUMBA_CUDA_PER_THREAD_DEFAULT_STREAM`.
```

Comment:
If both are set by accident, what happens?

Reply:
That may not be an accident - suppose you're using RMM (which always uses the NVIDIA bindings) and Numba with the ctypes bindings, and you want both to use PTDS, then that's what you should do. If you try to use `NUMBA_CUDA_PER_THREAD_DEFAULT_STREAM` when Numba is using the NVIDIA bindings, you will get a warning though, because that won't have the effect you want.

Reply:
Many thanks for the clarification.